jooby/src/main/java/org/jooby/internal/LocaleUtils.java

```diff
@@ -50,10 +51,14 @@ public static Locale parseOne(final String value) {
   }
 
   public static List<Locale.LanguageRange> range(final String value) {
-    List<Locale.LanguageRange> range = Locale.LanguageRange.parse(value);
-    return range.stream()
-        .sorted(Comparator.comparing(Locale.LanguageRange::getWeight).reversed())
-        .collect(Collectors.toList());
+    try {
+      List<Locale.LanguageRange> range = Locale.LanguageRange.parse(value);
+      return range.stream()
+          .sorted(Comparator.comparing(Locale.LanguageRange::getWeight).reversed())
+          .collect(Collectors.toList());
+    } catch (IllegalArgumentException e) {
+      return Collections.emptyList();
```

Comment:
This was to allow absent or erroneous `Accept-Language` (see three first cases of `Issue273.noMatch()`). Do you think that only the absent case should be handle?